frontend/apps/marketing/src/components/heading/Heading.tsx

```diff
@@ -0,0 +1,65 @@
+import '@code-dot-org/component-library/typography/index.css';
+import {
+  default as Typography,
+  VisualAppearance,
+  SemanticTag,
+} from '@code-dot-org/component-library/typography';
+import React, {ReactNode} from 'react';
+
+type HeadingVisualAppearance = Exclude<
```

Comment:
Rather than using `Exclude`, could we use `Pick` for only the ones we want? If we add a new one it may show up unexpectedly.